Disable SSL peer verification

From: Christian Prochaska <christian.prochaska@genode-labs.com>

With enabled peer verification, currently often a 'handshake error'
occurs. This patch disables the peer verification until a fix for
the handshake problem has been found.
---
 src/network/ssl/qsslconfiguration_p.h |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/network/ssl/qsslconfiguration_p.h b/src/network/ssl/qsslconfiguration_p.h
index 9c634f2..62ae400 100644
--- a/src/network/ssl/qsslconfiguration_p.h
+++ b/src/network/ssl/qsslconfiguration_p.h
@@ -81,7 +81,11 @@ class QSslConfigurationPrivate: public QSharedData
 public:
     QSslConfigurationPrivate()
         : protocol(QSsl::SecureProtocols),
+#ifdef Q_OS_GENODE
+          peerVerifyMode(QSslSocket::VerifyNone),
+#else
           peerVerifyMode(QSslSocket::AutoVerifyPeer),
+#endif
           peerVerifyDepth(0),
           sslOptions(QSsl::SslOptionDisableEmptyFragments
                      |QSsl::SslOptionDisableLegacyRenegotiation
